lib/coffee/AccessTokenEncryptor.coffee

```diff
@@ -18,6 +22,9 @@ class AccessTokenEncryptor
 		throw Error("cipherPassword too short") if @cipherPassword.length < 16
 
 	encryptJson: (json, callback) ->
+		unless ["2015.1", "2016.1"].includes(@cipherLabel)
```

Comment:
I don't think this needs to be conditional.  We can always write encrypt the password in the new format (because the format and the label are orthogonal).  Even if we have an old label in use we can still store the encrypted result in the new format - so we only need to encrypt in the new format.

Reply:
Right, yes I've been thinking about the label incorrectly. This makes sense.